README.md

```diff
@@ -261,7 +261,7 @@ This strategy assumes that each migration script in the `db` directory has a u
 
    ```
 
-   PG_PORT=<your db's port> PG_HOST=<your db's host> PG_PASSWORD=<your db's password> PG_USER=<your db's user> PG_DATABASE=<your db name> setup_postgresql.sh
+   PG_PORT=<your db's port> PG_HOST=<your db's host> PG_PASSWORD=<your db's password> PG_USER=<your db's user> PG_DATABASE=<your db name> ./setup_postgresql.sh
```

Comment:
could have sworn this was already in another PR and that I had said "ha, finally!"🤔

Reply:
You did! But it was the one about `./setup_postgres_local.sh` . So we'd missed it in both places.